source/sass/blocks/variables.scss

```
@@ -1,4 +1,15 @@
+// Viewports
+
+$tablet-width: 768px;
+$desktop-width: 1440px;
```

Стоит изменить значение:
```suggestion
$desktop-width: 1300px;
```